### PR TITLE
WIP: New SSL info

### PR DESCRIPTION
**Pull request by lucperkins**
@macintux Could I get a quick sanity check from you?

### CODE REVIEW

**Comment by lucperkins** [comment]
@angrycub Ready for review

File: source/languages/en/riak/ops/running/authz.md [hunk 1]
```diff
@@ -749,6 +749,38 @@ installing it can be found in [[Installing Erlang]]. This issue should
...
+## Enabling SSL
+
+In order to use any authentication or authorization features, you must
+enable SSL for Riak. **SSL is disabled by default**. If you are using
+[[Protocol Buffers|PBC API]] as a transport protocol for Riak (which we
+strongly recommend), enabling SSL on a given node requires only that you
+specify a [[host and port|Configuration Files#Client-Interfaces]] for
+the node as well as a [[certificate configuration|Authentication and
+Authorization#Certificate-Configuration]]. If you [[enable
+security|Authentication and Authorization#Security-Basics]]
```
Comment (macintux) [inline source/languages/en/riak/ops/running/authz.md]: Missing half the sentence?

**Comment by macintux** [comment]
This feels like an incomplete rework of the security checklist in this document. What purpose does it serve?

File: source/languages/en/riak/ops/running/authz.md (hunk 1 above)
Comment (lucperkins) [inline source/languages/en/riak/ops/running/authz.md]: @macintux Just noticed that. FIxed.

**Comment by lucperkins** [comment]
@macintux TBQH, I needed something to link to in other documents that talk about enabling SSL. That's really it. I just wanted to make sure that I'm not saying anything off base.

**Pull request merged**